content/devdocs/references/p2p_networking.md

```diff
@@ -108,7 +113,63 @@ The following annotated hexdump shows a `GIVB` message.  (The
 message header has been omitted.)
 
 {{ <highlight text> }}
-TODO : GIVB message hex dump
+GiveBlocksMessage:
+```
+8a 01 00 00 00 00 00 00 02 00 00 00 00 00 00 00 
```

Comment:
No need to add full message . It's too much , especially if messages are long. Annotated message would be enough. Please remove